src/runner.py

```diff
@@ -144,6 +143,7 @@ def export_cleaning(self, df, open_data):
             )
         if open_data:
             df = open_data_recode(self.name, df, self.headers)
+            df.to_csv(f"{self.output_file}_after_recode.csv", index=False)
```

Comment:
Not sure about writing to `.output` as the convention we use is generally that files in `.output` as sent to DO. Can we send this to another .gitignore folder called like `.cache` or something like that? And include in-line comment that we cache dataset for checking?